GUI.py

```diff
@@ -1,128 +1,905 @@
+        # streamlabs_polly_voice
+        config ["settings"]["tts"]["streamlabs_polly_voice"] = self.tts_streamlabs_polly_voice.get()
+
+        # tiktok_voice
+        config ["settings"]["tts"]["tiktok_voice"] = self.tts_tiktok_voice.get()
+
+        # python_voice
+        config["settings"]["tts"]["python_voice"] = self.tts_python_voice.get()
+
+        # py_voice_num
+        config["settings"]["tts"]["py_voice_num"] = self.tts_py_voice_num.get()
+
+        # silence_duration
+        config["settings"]["tts"]["silence_duration"] = self.tts_silence_duration.get()
+
+    # Background settings
+
+        # background choice (For builtin)
+        config["settings"]["background"]["background_choice"] = self.background_builtin.get()
+
+        # background typ
+        config["settings"]["background"]["background_type"] = self.background_select.get()
+
+        #TODO Add support for youtube url and custom backgrounds
+
+    # Dump new data into config.toml
+        configWrite = open("config.toml", "w")
+        toml.dump(config, configWrite)
+
+        print("Settings saved to config.toml")
+
+    # Background file browse
+    def backgroundBrowse(self):
+        filepath = filedialog.askopenfilename(
+            initialdir="/",
+            title="Select a mp4 file",
+            multiple=False,
+            filetypes=(("mp4 files", "*.mp4"), ("all files", "*.*"))
+        )
+        filename = Path(filepath).stem
+
+        self.background_custom.configure(text=filename)
+
+        if filepath == "":
+            self.background_custom.configure(text="Select background")
+        else:
+            #get current path
+            current_path = os.getcwd()
+            Path("./assets/backgrounds/").mkdir(parents=True, exist_ok=True)
+            shutil.copy(filepath, f"{current_path}\\assets\\backgrounds\\{filename}.mp4")
+
+    # Background type switch
+    def switchBackgroundType(self, type):
+        if type == "custom":
+            self.background_custom_title.tkraise()
+            self.background_custom.tkraise()
+
+        if type == "builtin":
+            self.background_builtin_title.tkraise()
+            self.background_builtin.tkraise()
+
+        if type == "youtube":
+            self.background_youtube_title.tkraise()
+            self.background_youtube.tkraise()
+
+    # tts voice change
+    def ttsVoiceChange(self, type):
+        if type == "streamlabspolly":
+            self.tts_streamlabs_polly_voice_title.tkraise()
+            self.tts_streamlabs_polly_voice.tkraise()
+
+        if type == "tiktok":
+            self.tts_tiktok_voice_title.tkraise()
+            self.tts_tiktok_voice.tkraise()
+
+        if type == "googletranslate":
+            self.tts_google_translate_voice_title.tkraise()
+            self.tts_google_translate_voice.tkraise()
+
+        if type == "awspolly":
+            self.tts_aws_polly_voice_title.tkraise()
+            self.tts_aws_polly_voice.tkraise()
+
+        if type == "pyttsx":
```

Comment:
Try not to hardcode this to each option

Reply:
Hi,
Thanks for reviewing our code! I understand it. We're working on it, but the tkraise can't be a variable. So it's difficult to do it. Do you have a suggestion for us on how we can do it?